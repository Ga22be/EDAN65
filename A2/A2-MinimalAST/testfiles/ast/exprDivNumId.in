int div() {
    int a;
    int b;
    b = 6;
    a = 12 / b;
}