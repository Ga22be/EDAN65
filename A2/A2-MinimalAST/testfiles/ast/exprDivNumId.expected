Program
  List
    Function
      IdDecl ID="div"
      List
        Declaration
          IdDecl ID="a"
        Declaration
          IdDecl ID="b"
        Assignment
          IdDecl ID="b"
          Numeral NUMERAL="6"
        Assignment
          IdDecl ID="a"
          Div
            Numeral NUMERAL="12"
            IdUse ID="b"
